build/kube_setup.sh

```diff
@@ -63,7 +63,7 @@ mkdir -p "$DOWNLOAD_DIR"
 
 
 function install_strimzi_operator {
-    STRIMZI_VERSION=0.22.1
+    STRIMZI_VERSION=0.25.0
```

Comment:
We went to v0.24.0 as of here: https://github.com/RedHatInsights/clowder/commit/a6d7f44c5218f2a6877e5cb151e2cbd4a38eda7c#diff-3f7aa29c9a9e65ce82ef5763d1a798a1aaf42b5edb119f4e7df1c9d44b0840bfR66